src/anjaliS/ArrayComparison.java

```diff
@@ -0,0 +1,17 @@
+package anjaliS;
+
+public class ArrayComparison {
+	public static void main(String args[]) {
+		int arr1[] = { 2, 4, 6, 8, 10 };
+		int arr2[] = { 2, 4, 6, 9, 10 };
+		boolean flag = true;
+		for (int index = 0; index < arr1.length; index++) {
+			if (arr1[index] != arr2[index]) {
+				flag = false;
+				System.out.println("Not Identical");
+				return;
+			}
+			System.out.println("Identical");
```

Comment:
You are making flag value false if condition matches but not using that, 
Refer this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/638/files